server/controllers/RunController.js

```diff
@@ -0,0 +1,41 @@
+const RunService = require('../services/RunService');
+
+async function configureRuns(req, res) {
+    const cycle = req.body.data;
```

Comment:
I'm guessing we won't want to use this word "cycle" -- but we are sending configuration data and then getting back all active runs right? So maybe it's just "config". This is a nit pick and maybe we just don't need to make this change now.